src/custom-rules/values.ts

```diff
@@ -1,10 +1,17 @@
+import { isEmpty } from "lodash";
```

Comment:
use sub-directory import to reduce import size

```suggestion
import isEmpty from "lodash/isEmpty";
```